Models/Player.cs

```diff
@@ -23,6 +23,8 @@ public class Player {
         public ObservableCollection<Item> items        { get; set; }
         public List<Slot>                 slots        { get; set; }
 
+        public int characterLevel => computeLevel(stats);
```

Comment:
Needs `[JsonIgnore]` else it'll write it out to the save file.

Reply:
Okay, thanks for checking that.